components/inputs/test/input-search.visual-diff.js

```diff
@@ -57,38 +42,4 @@ describe('d2l-input-search', () => {
 		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 	});
 
```

Comment:
I ended up deleting these tests. First, they were relying on querying the element's shadow root in order to focus -- which `focusWithKeyboard()` doesn't support. Second, they're essentially redundant because they're testing the focus/hover styles of `<d2l-button-icon>`, which we already have tests for.

Reply:
Is this comment out of date? Looks like the tests are restored.

Reply:
Oh yup -- after we added the ability to pierce shadow root's I ended up restoring them. While it's true that they're testing button-icon things, they're also somewhat testing the position of that focus-ring'd button-icon inside the search box. So kind of useful still.